tests/test_info.py

```diff
@@ -195,3 +195,14 @@ def test_is_draft_pull_request(fake_github, title):
     # No matter what the title, a pr is Draft if it says it is.
     pr = fake_github.make_pull_request(title=title, draft=True)
     assert is_draft_pull_request(pr.as_json())
+
+def test_check_csv_users_only():
+    people = get_people_file()
+    user = 'Carlos-Muniz'
+    # This user exists in the yaml but not in the csv, and should not be in people
+    assert people.get(user) == None
+
+def test_check_csv_org_priority():
+    people = get_people_file()
+    user = 'mmprandom'
+    assert people.get(user).get('agreement') == 'individual'
```

Comment:
Here's another place where `[]` is better than `.get()` (twice).

Reply:
Changed.